gtsfm/frontend/retriever/default_retriever.py

```diff
@@ -0,0 +1,50 @@
+"""Default image retriever for front-end.
+
+Authors: Travis Driver
+"""
+
+# import abc
+from typing import Optional
+
+
+from gtsfm.frontend.retriever.retriever_base import RetrieverBase
+import gtsfm.utils.logger as logger_utils
+from gtsfm.two_view_estimator import TwoViewEstimator
+
+
+logger = logger_utils.get_logger()
+
+
+class DefaultRetriever(RetrieverBase):
```

Comment:
thanks for the PR, @travisdriver. I think we should name this as `TemporalRetriever`, so that the name is a bit more descriptive

Reply:
It doesn't really use any temporal information. Maybe `SequentialRetriever` or `SlidingWindowRetriever`?

Reply:
True, we can't guarantee the sequence was captured in chronological order. I like the term "SequentialRetriever". If I remember correct, that's also the terminology COLMAP uses.